.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: Run tests
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+      with:
+        python-version: "3.7"
+    - name: Test
+      run: |
+        cd tests
+        make test
```

Comment:
Looks like the `test` target in the Makefile in the root directory does this, so you might be able to do something like `run: make test`